xml/System.Net.NetworkInformation/IPStatus.xml

```diff
@@ -54,6 +54,7 @@
   
  :::code language="cpp" source="~/snippets/cpp/VS_Snippets_Remoting/NclPingSync/CPP/syncping.cpp" id="Snippet1":::
  :::code language="csharp" source="~/snippets/csharp/System.Net.NetworkInformation/IPStatus/Overview/syncping.cs" id="Snippet1":::  
+ :::code language="fsharp" source="~/snippets/fsharp/System.Net.NetworkInformation/IPStatus/Overview/syncping.cs" id="Snippet1":::  
```

Comment:
Copying is the root of all evil :)

```suggestion
 :::code language="fsharp" source="~/snippets/fsharp/System.Net.NetworkInformation/IPStatus/Overview/syncping.fs" id="Snippet1":::  
```

Reply:
Thanks, these got me.  I wasn't about to find out to check the snippets locally, so I was trying out the PR to see if the CI would catch problems.  I wasn't expecting anyone to be fast on the review :)